Sources/SwiftDocCUtilities/PreviewServer/PreviewHTTPHandler.swift

```diff
@@ -54,26 +53,20 @@ final class PreviewHTTPHandler: ChannelInboundHandler {
     private var handlerFuture: EventLoopFuture<Void>?
     private let fileIO: NonBlockingFileIO
 
-    // When serving content on local network over SSL we authorize users via these credentials
-    private let credentials: (user: String, pass: String)?
-    
     /// - Parameters:
     ///   - fileIO: Async file I/O.
     ///   - rootURL: The root of the content directory to serve.
     ///   - credentials: Optional user credentials to authorize incoming requests.
-    public init(fileIO: NonBlockingFileIO, rootURL: URL, credentials: (user: String, pass: String)? = nil) {
+    init(fileIO: NonBlockingFileIO, rootURL: URL) {
```

Comment:
question: did this not need to be public?

Reply:
Oh good question- sorry I meant to call this out in the PR. The parent class is `internal` so this was effectively already `internal.